flow-defs/__types__.js.flow

```diff
@@ -0,0 +1,307 @@
+/* eslint-disable filenames/match-regex */
```

Comment:
can we avoid having this file here?

Reply:
No without updating ts-to-flow. Created: https://jira.tid.es/browse/WEB-423